src/bin/rq.rs

```diff
@@ -32,7 +32,7 @@ See https://github.com/dflemstr/rq for in-depth documentation.
 
 Usage:
   rq (--help|--version)
-  rq [-j|-a|-c|-h|-m|-p <type>|-t|-y] [-J|-A <type>|-C|-H|-M|-P <type>|-T|-Y] [--format <format>] [-l <spec>|-q] [--trace] [--] [<query>]
+  rq [-j|-r|-a|-c|-h|-m|-p <type>|-t|-y] [-J|-R|-A <type>|-C|-H|-M|-P <type>|-T|-Y] [--format <format>] [-l <spec>|-q] [--trace] [--] [<query>]
```

Comment:
These flags are alphabetical except for `-j` which is the default, can we match that convention? (Also below)